forgerock-auth/src/main/java/org/forgerock/android/auth/callback/DeviceBindingCallback.kt

```diff
@@ -207,6 +211,7 @@ open class DeviceBindingCallback : AbstractCallback, Binding {
      * @param listener The Listener to listen for the result
```

Comment:
Please help to remove this "listener" doc

Reply:
done